Light_Install.rst

```diff
@@ -0,0 +1,68 @@
+**लाइट इनस्टॉल गाइड**
+
+मैटरमोस्ट एक मैसेजिंग और कोलैबोरेशन मंच है। मैटरमोस्ट के द्वारा,आप प्रतिदिन इस्तेमाल किए जाने वाले उपकरण एक ही जगह पर एकीकृत कर सकते है और कोई भी  अधिसूचना या कार्य नहीं छूटेगा।
+
+आरंभ करने से पहले आपको इनकी आवश्यकता होगी:
+- रूट लेवल एक्सेस वाला उबंटू सर्वर (18.04 या 20.04)
+- (रेकमेंडेड) आपके सर्वर से जुड़ा एक डोमेन नाम (यानी mymattermostserver.com)
+- (रेकमेंडेड) अपनी टीम के सदस्यों के ईमेल ताकि आप उन्हें सर्वर पर आमंत्रित कर सकें
+
+
+ **इंस्टालेशन**
+ -----------
+ 
+[मैटरमोस्ट ओमनीबस](https://docs.mattermost.com/install/mattermost-omnibus.html)  के द्वारा मैटरमोस्ट इनस्टॉल  करें।
+
+टर्मिनल विंडो खोलें और एंटेर करें:
+``curl -o- https://deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+फिर,मैटरमोस्ट ओमनीबस इनस्टॉल करने  लिए ये चलाएं:
+``sudo apt install mattermost-omnibus``
+
+
+!नोट 
+यद्यपि ओमनीबस को इनस्टॉल करने और कॉन्फ़िगर करने का अनुशंसित तरीका SSL इनेबल्ड है, यदि आपको उसके बिना (या डोमेन नाम के बिना) उपयोग या परीक्षण करना चाहते हैं, तो ये चलाएं: 
```

Comment:
Anushashit means discipled/ordered AFAIU. Not "recommended". Is there a better word we can use here?

Reply:
This word is actually 'Anushancit', not 'Anushacit' and it means recommended. Though, updated it to avoid any discrepancy.

Reply:
I have used the word 'Suncit' instead of it.

Reply:
Ah yes, thanks!